apollo-router-core/src/query_planner/mod.rs

```diff
@@ -200,8 +216,9 @@ impl PlanNode {
             Self::Sequence { nodes } | Self::Parallel { nodes } => {
                 Box::new(nodes.iter().flat_map(|x| x.service_usage()))
             }
-            Self::Fetch(fetch) => Box::new(vec![fetch.service_name.as_str()].into_iter()),
-            Self::Flatten(flatten) => Box::new(flatten.node.service_usage()),
+            Self::Fetch(fetch) => Box::new(Some(fetch.service_name()).into_iter()),
+
+            Self::Flatten(flatten) => flatten.node.service_usage(),
```

Comment:
:yarn: space consistency fix proposal

```suggestion
            Self::Fetch(fetch) => Box::new(Some(fetch.service_name()).into_iter()),
            Self::Flatten(flatten) => flatten.node.service_usage(),
```